syntax = "proto2";

package lightdb.serialization;

message TileConfiguration {
    required uint32 version = 1;

    required uint32 numberOfColumns = 2;
    required uint32 numberOfRows = 3;
    repeated uint32 widthsOfColumns = 4;
    repeated uint32 heightsOfRows = 5;
}